package.json

```diff
@@ -33,9 +33,20 @@
     "build:umd:min": "cross-env BABEL_ENV=rollup NODE_ENV=production rollup -c -o dist/react-redux.min.js",
     "build": "npm run build:commonjs && npm run build:es && npm run build:umd && npm run build:umd:min",
     "clean": "rimraf lib dist es coverage",
-    "lint": "eslint src test",
+    "lint": "eslint src test/utils test/components test/getTestDeps.js",
     "prepare": "npm run clean && npm run build",
+    "pretest": "node ./install-test-deps.js",
     "test": "jest",
+    "test:ci": "node ./run-tests.js",
+    "pretest:watch": "node ./install-test-deps.js",
+    "test:watch": "jest --watch",
```

Comment:
For similar reasons, I would also remove this. `npm test -- --watch` is equivalent and what I do today.

Reply:
ok